Problem opening several dialogs on debian testing using wx3.0

E.g. open voxelplan
```
Traceback (most recent call last):
  File "/home/bassler/Projects/pytripgui/pytripgui/main.py", line 316, in voxelplan_load_dialog
    message="Choose headerfile")
  File "/usr/lib/python2.7/dist-packages/wx-3.0-gtk2/wx/_windows.py", line 3139, in __init__
    _windows_.FileDialog_swiginit(self,_windows_.new_FileDialog(*args, **kwargs))
PyAssertionError: C++ assertion "Assert failure" failed at ../src/common/filefn.cpp(1685) in wxParseCommonDialogsFilter(): missing '|' in the wildcard string!

```

--- a/pytripgui/main.py
+++ b/pytripgui/main.py
@@ -300,7 +300,7 @@ class MainFrame(wx.Frame):
         dlg = wx.FileDialog(
             self,
             defaultFile=self.tripexec_path,
-            wildcard="TRiP Exec File (*.exec)|*.exec|",
+            wildcard="TRiP Exec File (*.exec)|*.exec",
             message="Choose TRiP Exec File")
         if dlg.ShowModal() == wx.ID_OK:
             data = PytripData()
@@ -312,7 +312,7 @@ class MainFrame(wx.Frame):
         dlg = wx.FileDialog(
             self,
             defaultFile=self.voxelplan_path,
-            wildcard="Voxelplan headerfile (*.hed)|*.hed|",
+            wildcard="Voxelplan headerfile (*.hed)|*.hed",
             message="Choose headerfile")
         if dlg.ShowModal() == wx.ID_OK:
             data_obj = PytripData()

